ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/ratings/ratings.html.dir/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:file">
+    <jcr:content
+        jcr:lastModifiedBy="Administrator"
+        jcr:mimeType="text/plain"
+        jcr:primaryType="nt:resource"/>
+</jcr:root>
```

Comment:
@nitheshkumar-hcl  Why we need this file?

Reply:
Removed the file as per the review comments